rules/rbe_repo.bzl

```diff
@@ -967,7 +987,7 @@ def rbe_autoconfig(
           Should not be set if 'digest' is used.
           Must be set together with 'registry' and 'repository'.
           Note if you use any tag other than 'latest' (w/o specifiyng 'base_container_digest')
-          configs will need to be generaed, and a container will need to be pulled.
+          configs will need to be generated, and a container will need to be pulled.
```

Comment:
Maybe split this into a separate commit.

Reply:
ok